README.md

```diff
@@ -32,46 +35,74 @@ includes:
-1. Call [Stedi EDI Translate](https://www.stedi.com/products/edi-translate) to transform the JSON payload into an EDI file.
-1. Look up configured destinations for the specific partnership and transaction set ID. Refer to [Destinations](#destinations) for details.
+The `edi-outbound` function performs the following steps when it receives a payload and a metadata object. The payload
+must match the shape of the Guide's JSON Schema for writing EDI. Or, if a mappingId is specified, then the payload is
+the input for a mapping which will output valid Guide JSON data.
+
+1. Use the metadata to look up the configuration values required to construct an EDI envelope. The `partnershipId` is
+   the only required field.
+1. If a [Stedi Mapping](https://www.stedi.com/products/mappings) is specified, apply the mapping transformation to the
+   JSON.
+1. Call [Stedi EDI Translate](https://www.stedi.com/products/edi-translate) to transform the JSON payload into an EDI
+   file.
+1. Look up configured destinations for the specific partnership and transaction set ID. Refer
+   to [Destinations](#destinations) for details.
 1. Send failures to [Execution Error Destinations](#execution-error-destinations).
-1. Retry function execution failures 2 or more times. These retries can result in destinations receiving multiple messages, so you must handle at-least-once message delivery separately. We recommend using payload control numbers and message timestamps for deduplication.
+1. Retry function execution failures 2 or more times. These retries can result in destinations receiving multiple
+   messages, so you must handle at-least-once message delivery separately. We recommend using payload control numbers
+   and message timestamps for deduplication.
 
 ### Processed functional groups workflow
 
-The `edi-acknowledgement` function listens to Stedi Core `functional_group.processed` inbound events, which contain the partnership, document direction, and envelope data for a single functional group. When it receives an event, the function performs the following steps:
+The `edi-acknowledgement` function listens to Stedi Core `functional_group.processed` inbound events, which contain the
+partnership, document direction, and envelope data for a single functional group. When it receives an event, the
+function performs the following steps:
 
-1. If the direction is `RECEIVED`, look up up 997 acknowledgment configuration for the specific partnership and transaction set Ids in the functional group. Refer to [Acknowledgments](#acknowledgment-destinations) for details.
-1. If transaction sets are in the functional group with 997 acknowledgments configured, generate a 997 EDI-like JSON file and send it to the `edi-outbound` function for processing.
+1. If the direction is `RECEIVED`, look up up 997 acknowledgment configuration for the specific partnership and
+   transaction set Ids in the functional group. Refer to [Acknowledgments](#acknowledgment-configuration) for details.
```

Comment:
🍖 this is the only actual change I made to the README (to fix a broken anchor link). the rest is automatic formatting updates via my IDE